### DISCUSSION TITLE
part-ii/2017-74/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#84] [Question comments]
Part II 2017 General Relativity: Paper 4, Section II, D 

https://questions.tripos.org/part-ii/2017-74/

**Comment by AndoBando** [comment]
Remember there is an implicit "Real part" whenever we use complex numbers. If you write the energy flux in terms of $I$ the (not reduced) quadrupole tensor, it makes things slightly cleaner. You wont have to deal with matrices.
